examples/Example/src/prot_snap/index.tsx

```diff
@@ -0,0 +1,347 @@
+      .simultaneousWith(buttonDoubleTapGesture)
+  );
+
+  const scaleGestureHandler = useAnimatedGesture(previewPinchGesture);
+
+  if (remainingTime <= 0) {
+    finishRecording();
+  }
+
+  function stopFilterScroll() {
+    filter.value = withTiming(updateSelectedFilter(), { duration: 200 });
+  }
+
+  function updateSelectedFilter() {
+    let targetFilter = Math.round(filter.value);
+    if (targetFilter < 0) targetFilter = 0;
+    if (targetFilter >= filters.length) targetFilter = filters.length - 1;
+    setSelectedFilter(targetFilter);
+
+    return targetFilter;
+  }
+
+  function takePhoto() {
+    Alert.alert('You took a photo');
+  }
+
+  function takeSeries() {
+    Alert.alert('You took a series of photos');
+  }
+
+  function startRecording() {
+    setRecording(true);
+    setRemainingTime(MAX_VIDEO_DURATION);
+    setRecordingInterval(
+      setInterval(() => {
+        setRemainingTime((r) => r - 200);
+      }, 200)
+    );
+  }
+
+  function finishRecording() {
+    setRecording(false);
+    clearInterval(recordingInterval);
+    setRemainingTime(MAX_VIDEO_DURATION);
+
+    Alert.alert(
+      'You took a video (' +
+        (MAX_VIDEO_DURATION - remainingTime) * 0.001 +
+        ' s)'
+    );
+  }
+
+  const zoomStyle = useAnimatedStyle(() => {
+    return { transform: [{ scale: zoom.value }] };
+  });
+
+  return (
+    <Animated.View style={styles.container}>
+      <GestureMonitor gesture={scaleGestureHandler}>
+        <Animated.View
+          style={[styles.home, { backgroundColor: filters[selectedFilter] }]}>
+          <Animated.View style={[styles.box, zoomStyle]} />
+        </Animated.View>
+      </GestureMonitor>
+
+      <GestureMonitor gesture={panGestureHandler}>
+        <Animated.View style={styles.buttonContainer}>
+          <FilterCarousel filters={filters} selected={filter} />
+          <GestureMonitor gesture={buttonGestureHandler}>
+            <CaptureButton progress={1 - remainingTime / MAX_VIDEO_DURATION} />
+          </GestureMonitor>
+        </Animated.View>
+      </GestureMonitor>
+    </Animated.View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+    backgroundColor: 'white',
+  },
+  home: {
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+    opacity: 0.5,
+  },
+  buttonContainer: {
+    width: '100%',
+    height: 100,
+    position: 'absolute',
+    bottom: 50,
+    zIndex: 10,
+  },
+  shutterContainer: {
+    width: 100,
+    height: 100,
+    alignSelf: 'center',
+  },
+  box: {
+    alignSelf: 'center',
+    margin: 100,
+    width: 200,
+    height: 200,
+    backgroundColor: 'black',
+  },
+});
+
+function FilterCarousel(props) {
+  return (
+    <Animated.View style={filterCarouselStyles.container}>
+      {props.filters.map((filter) => {
+        return (
+          <Filter key={filter} filter={filter} selected={props.selected} />
+        );
+      })}
```

Comment:
You can use inline lambda here:
```ts
xs.map(x => x)
```

and
```ts
xs.map(x => {
	return x;
})
```
are the same and the former is less verbose.

Reply:
Fixed in 335f9303c53b5821384184ff90a78331cfed9329